Library/Formula/bash.rb

```diff
@@ -6,16 +6,69 @@ class Bash < Formula
 
   stable do
     url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz"
-    mirror "https://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz"
```

Comment:
It would be one thing if the generic mirror-assigner url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz" could be listed here more than once, but immediately falling back to the canonical url when one mirror isn't feeling perky isn't exactly trying.